docs/tutorials/grover_with_sampler.ipynb

```diff
@@ -1,562 +1,402 @@
 {
```

Comment:
<p>I think this is nice and pedagogical. It might be worth pointing out that Qiskit has tooling to simplify construction of classical oracles using a code snippet like what is found in the example here:</p><p> </p><p>https://qiskit.org/documentation/apidoc/classicalfunction.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Or I suppose you could just pass that `mark_circuit` as the oracle to `GroverOperator`.